evap/static/ts/src/csrf-utils.ts

```diff
@@ -19,8 +22,9 @@ function isMethodCsrfSafe(method: string): boolean {
 
 // setup ajax sending csrf token
 $.ajaxSetup({
-    beforeSend: function(xhr: JQuery.jqXHR, settings: JQuery.AjaxSettings) {
-        const isMethodSafe = settings.method && isMethodCsrfSafe(settings.method);
```

Comment:
Does it have a ~80 chars line length limit? If so, do we want to allow lines to be a bit longer? In python, we allow 120.